### DISCUSSION TITLE
OS should install without Internet.

### DISCUSSION THREAD

**Discussion by xialvjun** [Nitrux/nitrux-bug-tracker#158] [General]
And it should work with most of its functions without Internet.

We can install Windows without Internet. And after install, we can use it to do most things: play video/audio, install softwares(if you have the software installer)....

[The installation process requires mandatory active internet connectivity via wired or wireless connections.](https://nxos.org/tutorial/how-to-install-nitrux/#:~:text=The%20installation%20process%20requires%20mandatory%20active%20internet%20connectivity%20via%20wired%20or%20wireless%20connections.) is arrogant.

**Comment by UriHerrera** [comment]
Internet access is required for the installer to complete the installation process. You are welcome to browse our repositories and look for the settings for Calamares to understand *why* it is a requirement. They are publicly available.

Internet connectivity is also necessary to update the distribution and acquire new software. Any installable software, AppImages, Flatpaks, 0install binaries, containers, etc., are unavailable locally, i.e., on your computer or the installation media. All that content is downloaded from remote servers that your computer connects to using the Internet.

Functionalities like the examples you mention, like playing audio and video, never require Internet access.

You seem to be confusing the requirement of Windows for Internet connectivity to ours, going by your erroneous assessment that Internet connectivity is "arrogant." They're not the same.
